client/foundries_events.go

```diff
@@ -0,0 +1,42 @@
+package client
```

Comment:
Is this an effort to split the `foundries.go` into submodules?

Reply:
not submodules, just files. this is big enough we could start doing a types.go, etc. the one file is just getting painful to edit